Добавлены header, partners и footer

JstUsername opened the pull request:
Адаптив для header немного изменен, по сравнению с макетом.

JstUsername committed:
feat(assets/header): добавлены иконки и логотип для компонента header

JstUsername committed:
fix(theme): добавлен fontWeight для body2

JstUsername committed:
fix(handleClickShowPassword): setShowPassword вынесен в onClick

JstUsername committed:
feat(handleKeyDown): добавлена функция для переключения чекбокса по нажатию на клавиши

JstUsername committed:
feat(StyledLink): добавлены стили для Link исправляющие ширину

JstUsername committed:
fix(InputLabel): заголовки для input реализованы с помощью label

JstUsername committed:
fix(letter-spacing): удалены свойства letter-spacing

JstUsername committed:
fix(assets/header): расширен набор иконок для компонента header

JstUsername committed:
fix(theme): добавлен default вариант для background

JstUsername committed:
feat(svgr): добавлен и настроен плагин svgr

JstUsername committed:
feat(constants.ts): добавлены константы для компонента Header

JstUsername committed:
feat(types.ts): добавлены типы для компонента Header

JstUsername committed:
feat(Header.tsx): добавлен компонент Header

JstUsername committed:
fix(App.tsx): компонент Header добавлен в App

JstUsername committed:
fix(constants, types): константы и типы вынесены в папку Header

JstUsername committed:
fix(Header, constants): исправлены импорты после переноса констант и типов

JstUsername committed:
fix(LiButton): рефакторинг свойства padding

JstUsername committed:
fix(LoginProps): исправлено название интерфейса

JstUsername committed:
fix(theme): исправлено использование theme в styled-components

# Conflicts:
#	src/components/Login.tsx

JstUsername committed:
fix(Login.tsx): компонент Login вынесен в отдельную папку

JstUsername committed:
fix(Login, App): исправлены импорты после переноса Login

# Conflicts:
#	src/App.tsx

JstUsername committed:
feat(Input.tsx): поля формы и их стили вынесены в отдельный компонент

JstUsername committed:
fix(Grid): стилизация Grid вынесена в styled-components

JstUsername committed:
feat(dependencies): в проект добавлен React Router

JstUsername committed:
feat(BrowserRouter): компонент App обернут в BrowserRouter

JstUsername committed:
feat(Routes, AppContent): добавлены пути, контент приложения вынесен в AppContent

# Conflicts:
#	src/App.tsx

JstUsername committed:
fix(routing): роутинг реализован с помощью React Router

JstUsername committed:
fix(routing, types): роутинг реализован с помощью React Router, удален тип headerProps

JstUsername committed:
feat(commons): компонент Input переименован в FormInput и вынесен в commons

JstUsername committed:
fix(Input): исправлен иморт, Input => FormInput

JstUsername committed:
fix(Login): компонент переименован в LoginPage

JstUsername committed:
fix(FormInput): исправлено название интерфейса и компонента

JstUsername committed:
feat(HomePage.tsx): добавлен компонент для размещения основного контента приложения

JstUsername committed:
fix(import, routing): исправлены импорты и пути для роутинга, AppContent заменен на HomePage

# Conflicts:
#	src/App.tsx

JstUsername committed:
fix(BrowserRouter): BrowserRouter вынесен из main в App

JstUsername committed:
fix(icons): иконки отображены с помощью svgr

JstUsername committed:
fix(login.webp): logn.webp импортирован как модуль, исправлен src

JstUsername committed:
fix(@media): все медиа запросы переписаны на theme.breakpoints.(down/up)

JstUsername committed:
fix(responsive h2): изменение размера h2, в зависимости от брейкпоинта sm вынесено в theme

JstUsername committed:
fix(StyledLink): StyledLink вынесен в компонент FormLink

JstUsername committed:
fix(StyledCheckbox): StyledCheckbox вынесен в компонент FormCheckbox

JstUsername committed:
feat(assets/login): добавлена иконка закрытого глаза

JstUsername committed:
feat(FormPasswordInput): инпут для пароля вынесен из LoginPage в отдельный компонент.

Для FormPasswordInput добавлена иконка закрытого глаза при видимом пароле.

JstUsername committed:
fix(EyeCloseIcon): увеличен размер EyeCloseIcon и добавлен отступ

JstUsername committed:
feat(validation): добавлена валидация email и пароля.

Email валидируется на входе и регистрации, пароль валидируеся только на регистрации.

JstUsername committed:
fix(HomePage): компонент Header добавлен в HomePage

JstUsername committed:
feat(assets/partners): добавлены логотипы для компонента Partners

JstUsername committed:
feat(Partners): компонент Partners реализован и добавлен в HomePage

JstUsername committed:
fix(theme): переопределена типизация темы, добавлен variant="formHeader"

JstUsername committed:
fix(gradient): градиент вынесен в тему

JstUsername committed:
fix(padding): исправлены отступы в FormInput и LoginPage

JstUsername committed:
fix(FormControl): FormControl вынесен в FormInput, для FormInput добавлен inputSx

JstUsername committed:
feat(utils/validate): функции для валидации email и пароль вынесены в utils/validate

JstUsername committed:
feat(FormEmailInput): добавлен отдельный компонент для email.

FormInput очищен от стейтов и условий, в FormPassword добавлен PasswordSubtitle.

JstUsername committed:
fix(file structure): страницы перемещены в pages, содержимое commons перемещено в components/form

JstUsername committed:
fix(import): исправлены импорты после перемещения commons, LoginPage, HomePage

JstUsername committed:
fix(HomePage): исправлены импорты в HomePage

JstUsername committed:
feat(assets/footer): добавлены иконки и логотипы для компонента Footer

JstUsername committed:
feat(divider2, h6): в тему добавлен новый цвет и вариант шрифта

JstUsername committed:
fix(Header): изменено название StyledUl на HeaderUl, исправлены паддинги для CategoriesButton, добавлен cursor: pointer

JstUsername committed:
feat(Footer): компонент Footer реализован и добавлен в HomePage

JstUsername committed:
fix(adaptive): доработана адаптивная верстка для Footer

JstUsername committed:
feat(flex-wrap): добавлен перенос логотипов при переполнении для Partners

JstUsername committed:
fix(constants, types): BottomHeaderItems переименован в HeaderListItems, icon в LiButtonProps сделан опциональным

JstUsername committed:
feat(HeaderMobile.tsx): добавлен компонент для мобильной версии хэдэра

JstUsername committed:
fix(Header): в Header добавлен HeaderMobile, добавлены DesktopHeaderWrapper и DesktopLogo, скорректированы стили списка

JstUsername committed:
feat(handlePageSection): onClick для LiButton вынесен в отдельную функцию

JstUsername committed:
feat(error): в тему добавлен error оттенок

JstUsername committed:
feat(badge): добавлен бэдж для корзины в HeaderMobile

JstUsername committed:
fix(ShoppingCartIcon): ShoppingCartIcon заменена на TagIcon

u1timaD commented on src/components/Header/HeaderMobile/HeaderMobile.tsx:
Вроде как не должно быть два хедера меняющихся через дисплей, то есть мысль такая, чтобы заменять контент внутри хедера, а не делать два.

JstUsername replied:
Вообще наверное лучше сделать не display: none, а полный unmount компонента, который скрыт, подумаю как лучше.

u1timaD replied:
ну как минимум у тебя контенер хедера должен быть один мне кажется, а внутренности это отдельные 2 компонента., но тут надо подумать, может где-то референс глянуть

JstUsername replied:
Да, думаю оптимальным будет делать условный рендеринг разных компонентов внутри тэга header.

u1timaD commented on src/components/Header/HeaderMobile/HeaderMobile.tsx:
```diff
@@ -0,0 +1,231 @@
+  }
+`;
+
+const DropdownMenu = styled('div')<DropdownMenuProps>`
```
блин)) почему тебе удобней стили держать в одном файле, можно же вынести в отдельный....)  но я так понимаю это опционально )

u1timaD commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,261 @@
+            <MapMarkerIcon title="Map marker" />
+            <span style={theme.typography.body2}>Москва</span>
+          </HeaderTopBlock>
+          <HeaderTopBlock style={{ marginLeft: 'auto' }} onClick={goToLogin}>
```
вроде инлайново стили не есть хорошо

JstUsername replied:
Да, но он тут один, а вынести его в styled я не могу, т.к. применяется только на одном элементе.

u1timaD commented on src/components/Header/Header.types.ts:
```diff
@@ -0,0 +1,12 @@
+  icon?: FunctionComponent<SVGProps<SVGSVGElement>> | undefined;
+}
+
+export interface HeaderLi {
```
забыл Props или Types

u1timaD commented on src/components/Partners/Partners.tsx:
```diff
@@ -0,0 +1,79 @@
+        <WavesmarathonLogo title={'WAVESMARATHON'} />
+        <span style={{ display: 'inline-flex' }}>
+          <LogoTitle style={{ fontSize: '18px', lineHeight: '1' }}>WAVES</LogoTitle>
+          <LogoTitle style={{ fontSize: '18px', lineHeight: '1', fontWeight: '400' }}>MARATHON</LogoTitle>
```
тоже инлайн и ещё в некоторых местах

JstUsername replied:
Тоже идет как бы "переписывание" styled, большинство стилей остается из LogoTitle, но вот некоторые пришлось поменять. Стараюсь больше 3 инлайнов не делать.

u1timaD replied:
тут больше вопрос хорошо ли так делать вообще) если нет, придётся искать альтернативу, если уместно в некоторых случаях типа этого, то вопросов нет)

u1timaD commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,318 @@
+      <nav>
+        <FooterMiddle>
+          <div>
+            <ListTitle>Для взрослых</ListTitle>
```
можно сделать 1 компонент и прокидывать объект с тайтлом списка и сами итемы списка, и продублировать три раза прокидывая разные данные, чтоб избавиться от всего полотна))

IvanKoigerov replied:
Согласен, можешь выносить

majishanred commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,318 @@
+      <FooterBottom>
+        <Copyright>Тестовая компания @ 2023. Все права защищены.</Copyright>
+        <nav>
+          <FooterUl style={{ flexDirection: 'row', marginTop: '0px' }}>
```
А зачем это в стайл писать, у тебя же компонент есть, перенеси туда.

JstUsername replied:
FooterUl используется по всему компоненту и в разных случаях там нужны немного разные стили. Базово используется margin: 16px 0 0 и flex-direction: column.
В случае с 125 строкой нужен flexDirection: 'row' и проще всего его переназначить с помощью inline стиля.

VeronikaBulaeva commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,318 @@
+import { styled } from '@mui/material/styles';
+import { useMediaQuery } from '@mui/material';
+import theme from '../../theme.tsx';
```
theme в компоненты не импортируем

JstUsername replied:
Используется для mediaQuery, не понял как без импорта использовать.

VeronikaBulaeva commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,318 @@
+          <StyledMailIcon title="Mail" />
+          <StyledInput
+            id="input-mail"
+            placeholder={useMediaQuery(theme.breakpoints.down('sm')) ? 'Подписаться...' : 'Подписаться на рассылку'}
```
здесь нужно тип Theme в дженерик прокинуть

IvanKoigerov replied:
Не правильная тема хуки использовать где-то кроме тела

VeronikaBulaeva commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,318 @@
+            id="input-mail"
+            placeholder={useMediaQuery(theme.breakpoints.down('sm')) ? 'Подписаться...' : 'Подписаться на рассылку'}
+          />
+          <StyledButton>Подписаться</StyledButton>
```
кнопка есть в компонентах, переиспользуй

VeronikaBulaeva commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,318 @@
+          <div>
+            <div style={{ marginBottom: '24px' }}>
+              <ListTitle>Контакты</ListTitle>
+              <FooterUl style={{ flexDirection: 'row', gap: '8px' }}>
```
от инлайн стилей уходим)
везде поправь)

JstUsername replied:
Все уже написали, но что с этим делать не понятно. Создавать 3 отдельных компонента для элементов, которые отличаются 1 - 2 стилями еще более странно чем использовать инлайны. Можно добавить пропсы, но внешний вид кода не особо поменяется.

VeronikaBulaeva commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,318 @@
+            <div>
+              <ListTitle>Присоединиться к нам</ListTitle>
+              <FooterUl style={{ flexDirection: 'row', marginTop: '8px' }}>
+                <FooterLi>
```
можешь создать массив соц сетей и промапить

VeronikaBulaeva commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,261 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import theme from '../../theme.tsx';
```
опять импорт theme

VeronikaBulaeva commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,261 @@
+          </HeaderTopBlock>
+          <HeaderTopBlock style={{ marginLeft: 'auto' }} onClick={goToLogin}>
+            <UserIcon title="User" />
+            <span style={theme.typography.body2}>Вход</span>
```
theme используем по-другому

VeronikaBulaeva commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,261 @@
+          <Search>
+            <StyledSearchIcon title="Search" />
+            <StyledInput id="input-search" placeholder="Поиск по ..." />
+            <CategoriesButton>
```
используй компонент кнопки

VeronikaBulaeva commented on src/components/Header/HeaderMobile/HeaderMobile.tsx:
```diff
@@ -0,0 +1,231 @@
+import MenuIcon from '../../../assets/header/icons/menu.svg?react';
+import SearchIcon from '../../../assets/header/icons/search.svg?react';
+
+interface HeaderMobileProps {
```
нужно вынести в файл с типами
ну и все интерфейсы нужно вынести)

VeronikaBulaeva commented on src/components/form/FormInput/FormInput.tsx:
```diff
@@ -0,0 +1,90 @@
+import { SxProps } from '@mui/system';
+import FormControl from '@mui/material/FormControl';
+
+interface FormInputProps {
```
выносим в типы

VeronikaBulaeva commented on src/pages/Login/LoginPage.tsx:
```diff
@@ -0,0 +1,156 @@
+import { useNavigate } from 'react-router-dom';
+import Typography from '@mui/material/Typography';
+import { styled } from '@mui/material/styles';
+import theme from '../../theme.tsx';
```
theme импортируешь

JstUsername committed:
pr(Footer): доступ к theme организован с помощью useTheme, указан дженерик <Theme>

JstUsername committed:
fix(FooterLi): нестилизованный компонент FooterLi заменен на li

JstUsername committed:
pr(LoginPage): доступ к theme организован с помощью useTheme

JstUsername committed:
pr(Header): вместо span использован Typography

JstUsername committed:
pr(HeaderLi): название интерфейса HeaderLi изменено на HeaderLiType

pr(HeaderLi): название интерфейса HeaderLi изменено на HeaderLiType

JstUsername committed:
pr(Header): HeaderTopBlock использует Box, style заменен на sx

JstUsername committed:
pr(style): во всех компонентах инлайн стили заменены на sx/styled

JstUsername committed:
Merge branch 'main' into header-footer

# Conflicts:
#	package-lock.json
#	package.json
#	src/App.tsx
#	src/components/form/FormCheckbox/FormCheckbox.tsx
#	src/components/form/FormPasswordInput/FormPasswordInput.tsx
#	src/pages/HomePage/HomePage.tsx
#	src/pages/Login/LoginPage.tsx
#	src/theme.tsx
#	vite.config.ts

JstUsername committed:
pr(types): interface всех компонентов вынесены в component.types.ts

JstUsername committed:
pr(StyledButton): за основу для кнопок взят /commons/button/ButtonLink.

Переписаны стили, добавлен ButtonText.

JstUsername committed:
pr(Header): добавлен условный рендеринг хэдэров, в зависимости от размера экрана

JstUsername committed:
pr(SocialMediaIcons): добавление иконок посредством .map.

Информация о иконках помещена в Footer.constants, типы определны в Footer.types.

JstUsername committed:
fix(HeaderListItems): для SVG в HeaderList добавлен title

JstUsername committed:
feat(constants): изменены названия переменных в .constants

JstUsername committed:
pr(FooterMiddleListItems): добавление ссылок в FooterMiddle, посредством .map.

Информация о ссылках помещена в Footer.constants, типы определны в Footer.types.

JstUsername committed:
fix(transition): исправлена анимация для LiButton компонента Header

JstUsername committed:
fix(headerListItems): исправлен импорт headerListItems

IvanKoigerov commented on src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,289 @@
+  height: 48px;
+  width: 260px;
+  margin: 0;
+  padding: 0 0 0 48px;
```
зачем так пишешь? определенную сторону тут опиши

JstUsername replied:
Там по-умолчанию добавляются падинги, которые не нужны. С помощью padding: 0 0 0 48px избавляюсь от них.
![image](https://github.com/JstUsername/team-landing/assets/79771449/071f340d-d997-4b08-9707-3d7ddfc320a8)

IvanKoigerov commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,263 @@
+
+  return (
+    <header>
+      {useMediaQuery(theme.breakpoints.up('sm')) ? (
```
тоже юзаем только в теле

IvanKoigerov commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,263 @@
+  const theme = useTheme<Theme>();
+
+  return (
+    <header>
```
А почему хедер не стилизируешь?  а ещё там же кусок должен быть стики

IvanKoigerov commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,263 @@
+    transition: none;
+  }
+`;
+
+const CategoriesButtonIcon = styled(ChevronDownIcon)`
+  position: absolute;
+  top: 12px;
+  right: 16px;
+  path {
+    fill: ${({ theme }) => theme.palette.text.secondary};
+  }
+`;
```
как-то не совсем круто так двигать элементы

IvanKoigerov commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,263 @@
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
```
row дефолт

IvanKoigerov commented on src/components/Header/HeaderMobile/HeaderMobile.tsx:
Обычно эту вещь делают просто в одном компоненте, чтобы не ререндерить полностью, но ладно, давай так оставим

IvanKoigerov commented on src/components/Partners/Partners.tsx:
```diff
@@ -0,0 +1,84 @@
+
+const PartnersSection = styled('section')`
+  display: flex;
+  gap: 16px;
+  row-gap: 32px;
```
gap 2 параметра можно передвать

IvanKoigerov commented on src/components/Partners/Partners.tsx:
```diff
@@ -0,0 +1,84 @@
+  width: 200px;
+  display: flex;
+  gap: 8px;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  ${({ theme }) => theme.breakpoints.down('md')} {
+    width: fit-content;
+    justify-self: center;
+    align-self: center;
+  }
+`;
```
чем отличается 
justify-content: center;
align-items: center;

justify-self: center;
align-self: center;

JstUsername replied:
В данном случае justify-content и align-items используются для выравнивания контента (логотипов и текста) внутри соответствующего им блока. justify-self используется для выравнивания блока с логотипом и текстом относительно родительского блока при md, т.к. родительский элемент становится grid. align-self не нужен.

JstUsername committed:
fix(ButtonLink): для ButtonLink добавлены margin и align-self

JstUsername committed:
fix(favicon): добавлен полный пакет фавиконов

JstUsername committed:
pr(useMediaQuery): использование useMediaQuery в теле компонента

JstUsername committed:
pr(HeaderUl): удален лишний стиль flex-direction

JstUsername committed:
pr(PartnersSection, PartnerItem): исправлен gap, удален align-self

JstUsername committed:
feat(scrollbar): добавлен кастомный скроллбар

JstUsername committed:
pr(FooterMiddleUl): список из FooterMiddle вынесен в отдельный компонент FooterMiddleUl

JstUsername committed:
fix(scrollbar): кастомный скроллбар применен ко всем дочерним элементам body

JstUsername committed:
pr(HeaderWrapper): header стилизован с помощью HeaderWrapper, установлен position: sticky

JstUsername committed:
Merge branch 'main' into header-footer

# Conflicts:
#	src/pages/HomePage/HomePage.tsx
#	src/theme.tsx

JstUsername committed:
fix(gradient): mainGradient заменен на blueGradient, согласно theme

JstUsername committed:
fix(assets, commons): assets и commons перенесены в src/assets|commons

JstUsername committed:
fix(import): исправлены импорты после переноса assets и commons

JstUsername committed:
fix(theme): из темы удалены лишние React, в Gradients использован blueGradient

JstUsername committed:
fix(Partners): flex-wrap активируется на экранах меньше 1024px

JstUsername committed:
fix(README.md): актуализирована информация

myaak commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,272 @@
+    );
+  };
+
+  const navigate = useNavigate();
```
Такие вещи объявляются обычно сверху компонента, до функций и т.д..
Обычно сначала хуки, константы, а потом функции

myaak commented on src/components/Header/Header.tsx:
```diff
@@ -0,0 +1,272 @@
+  );
+}
+
+const HeaderWrapper = styled('header')`
+  position: sticky;
+  top: 0;
+  background: ${({ theme }) => theme.palette.background.default};
+  z-index: 9;
+  ${({ theme }) => theme.breakpoints.down('sm')} {
+    position: static;
+  }
+`;
```
В каких-то местах лежит в файле styled.
Тут очень много стайледа, почему тогда не вынесли

myaak commented on src/components/PartnerForm/PartnerFormField/PartnerFormField.styled.ts:
```diff
@@ -1,5 +1,5 @@
 import { styled } from '@mui/material';
-import chevronDown from '../../assets/PartnerForm/chevron-down.svg';
+import chevronDown from '../../../assets/PartnerForm/chevron-down.svg';
```
По хорошему чтобы таких вещей не было нужно задефайнить пути в tsconfig.
Обычно дефайнят на чето типа
`@` для scr
`~` для external вещей, можно для ассетов

IvanKoigerov replied:
Норм но уже не в этом пр

u1timaD replied:
Ну до лекции по тс я не знал про это)

myaak commented on src/components/Partners/Partners.tsx:
```diff
@@ -0,0 +1,84 @@
+        <LogoTitle variant="h4">ArtVenue</LogoTitle>
+      </PartnerItem>
+      <PartnerItem>
+        <KontrastrLogo title={'kontrastr'} />
```
Где-то пишете`title={"string"}`
Где-то `title="string"`

По хорошему выбрать один стиль и eslint на это настроить

JstUsername committed:
pr(const): переменные вынесены перед функциями

JstUsername committed:
pr(styled): все styled-components вынесены в styled.ts

JstUsername committed:
pr(eslint): добавлен eslint-plugin-react, в конфигурации настроен jsx-curly-brace-presence

JstUsername committed:
pr(curly-brace): для всех  JSX свойств использованы фигурные скобки

IvanKoigerov commented on src/components/ArticlesItem/ArticleItem.tsx:
```diff
@@ -8,21 +8,21 @@ import Loader from '../Loader/Loader';
-        <Typography variant="articleTitle" mb="16px">
+      <Box padding={"24px 16px 16px"}>
+        <Typography variant={"articleTitle"} mb={"16px"}>
           {title}
         </Typography>
-        <Typography variant="articleDescription">{description}</Typography>
+        <Typography variant={"articleDescription"}>{description}</Typography>
       </Box>
-      <Box padding="16px" width="100%" marginTop="auto">
-        <Link href="#" underline="none" display="inline-flex" alignItems="center" sx={{ cursor: 'pointer' }}>
-          <Typography variant="articleLinkText" textTransform="capitalize">
+      <Box padding={"16px"} width={"100%"} marginTop={"auto"}>
+        <Link href={"#"} underline={"none"} display={"inline-flex"} alignItems={"center"} sx={{ cursor: 'pointer' }}>
+          <Typography variant={"articleLinkText"} textTransform={"capitalize"}>
             Подробнее
           </Typography>
-          <Box display="flex" padding="5px" marginLeft="16px">
+          <Box display={"flex"} padding={"5px"} marginLeft={"16px"}>
             <ArrowRightSvg />
```
все хуйня)
так не делают обычно)
обычно описывают prop="значение"
нуууууууу ща не фиксим

IvanKoigerov reviewed (commented):
Заливай пр

JstUsername committed:
fix(scrollbar): удалены лишние стили для скроллбара

JstUsername committed:
fix(eslint): исправлена конфигурация jsx-curly-brace-presence

JstUsername committed:
fix(curly-brace): для строковых пропов удалены фигурные скобки

JstUsername committed:
fix(MedicalInformation): удален лишний marginBlock для MedicalInformation
